calico_containers/pycalico/netns.py

```diff
@@ -51,6 +52,49 @@ def setup_logging(logfile, level=logging.INFO):
     _log.addHandler(handler)
 
 
+def increment_metrics(namespace):
+    """
+    If any default route has a metric of 0, increase the metric of 
+    all default routes by 1.
+    :param namespace: The Networking namespace of the container.
+    :return: None. Raises CalledProcessError on error.
+    """
+    class Route(object):
+        def __init__(self, route_output):
+            self.route_output = route_output
+            self.default = route_output.startswith("default")
+            match = re.search('metric\s+(\d+)', route_output)
+            try:
+                self.metric = int(match.group(1))
```

Comment:
It's more traditional to check if match is None rather than trying it and catching an AttributeError.

Python has a ternary conditional operator too if you'd prefer to get this on one line:
```self.metric = int(match.group(1)) if match else 0```